docs/userguide.md

```diff
@@ -171,26 +172,31 @@ We support the following archive formats:
 - `.jar`
 - `.rpm`
 
-### Manually specifying dependencies
+## Manually specifying dependencies
 
 FOSSA offers a way to manually upload dependencies provided we support the dependency type. Manually specifying dependencies is very helpful in the event your package manager is unsupported or you are using a custom and nonstandard dependency management solution.
 
-The FOSSA CLI will automatically read `fossa-deps.yml` in the root directory when `fossa analyze` is run and parse dependencies from it.
+The FOSSA CLI will automatically read `fossa-deps.yml` in the root directory (usually the current working directory) when `fossa analyze` is run and parse dependencies from it.
+The CLI also accepts `fossa-deps.yaml`, but will abort if it finds both.
 
 > Tip: Use a script to generate this file before running `fossa analyze` to keep your results updated.
 
+To manually specify a dependency, you must provide the package type, package name, and optionally a package version, under the `referenced-dependencies` array, as shown here:
+
 ```yaml
-dependencies:
+referenced-dependencies:
 - type: gem
-  package: iron
+  name: iron
 - type: pip
-  package: Django
+  name: Django
   version: 2.1.7
 ```
-The `package` and `type` fields are required and specify the name of the dependency and where to find it. The `version` field is optional and specifies the preferred version of dependency.
+
+The `package` and `type` fields are required and specify the name of the dependency and where to find it. The `version` field is optional and specifies the preferred version of dependency.  Note that many version numbers can look like actual numbers in YAML, but they should always be strings!  You can quote them to make sure YAML treats them as strings, otherwise you may run into an error.
```

Comment:
> You can quote them to make sure YAML treats them as strings, otherwise you may run into an error

I ran into this issue in the `yarn.lock` PR, which resulted creating the the `TextLike` type in `Data.Aeson.Extra`. It allows us to parse accidental ints/doubles as `Text`, which would prevent us from pushing this obligation to the user